cms/envs/common.py

```diff
@@ -176,6 +176,9 @@
     # in sync with the ones in lms/envs/common.py
     'ENABLE_DISCUSSION_SERVICE': True,
     'ENABLE_TEXTBOOK': True,
+
+    # .. documented_elsewhere_name: ENABLE_STUDENT_NOTES
+    # .. documented_elsewhere:
```

Comment:
Is `documented_elsewhere` left over or a future `TODO`?

Reply:
It's a "takes no arguments" flag, actually, though the required colon makes that unclear.  Possibly just the `documented_elsewhere_name` field would make that obvious enough.